srcpkgs/castor/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'castor'
+pkgname=castor
+version=0.8.16
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+hostmakedepends="pkg-config cargo rust atk-devel pango-devel gdk-pixbuf-devel gtk+3-devel cairo-devel libressl-devel"
+makedepends="${hostmakedepends}"
```

Comment:
And then you should be able to move the `-devel` stuff to `makedepends` only, and remove cargo and rust as hostmakedepends, since cargo build style pulls them in.